scalingo.json

```
@@ -40,5 +40,8 @@
       "value": "kibana-password",
       "required": true
     }
-  }
+  },
+  "addons": [
+    {"plan": "elasticsearch:elasticsearch-sandbox"}
```

Sandbox is too limited to do anything, we should use the 512
```suggestion
    {"plan": "elasticsearch:elasticsearch-512"}
```

---

Okay thanks